flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala/StreamTableExample.scala

```diff
@@ -44,12 +44,12 @@ object StreamTableExample {
     val orderA = env.fromCollection(Seq(
```

Comment:
nit: 2nd commit
> [FLINK-8577][table] Rename methods name to toTableFromAppendStream and toTableFromUpsertStream in DataStreamConversions

has no code changes related to `toTableFromUpsertStream`. Probably commit should be renamed.

Reply:
Will rename it to `Rename methods name to toTableFromAppendStream in DataStreamConversions`